package.json

```diff
@@ -25,7 +25,7 @@
     "netlify"
   ],
   "engines": {
-    "node": ">=10.18.0"
+    "node": ">=12.20.0"
```

Comment:
```suggestion
    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

Reply:
This sounds good to me.

However, since `netlify-headers-parser` is a dependency of `netlify-cli`, the `engines.node` field of `netlify-cli` would also need to be updated accordingly:

https://github.com/netlify/cli/blob/08323afe67aa0f9a70a2dadb167ea682f0400353/package.json#L14

This would mean dropping Node 13, Node 14.0.0-14.13.0 and Node 15 support, so we would need to make a new announcement.

What do you think?

Also cc @JGAntunes

Reply:
I know sadly I recognized it too late once we already shipped it. Not sure if it is worth the effort though.

But this would be the propper way.

Reply:
> This would mean dropping Node 13, Node 14.0.0-14.13.0 and Node 15 support in Netlify CLI, which is a breaking change, so we would need to make a new major release and announcement.
>
> What do you think?

IMO if we opt for this approach we should go with another major release, semver numbers are cheap/breaking people's setup because we want to save on numbers is worse. Not sure if an announcement is required? (Assuming an announcement is a community post?) Maybe an update to the original announcement with an **Edit** would suffice.

Reply:
If we do the ESModules migration on the CLI we need a breaking version as well or? Maybe do it together?

Reply:
After discussing with @lukasholzer and @JGAntunes on Slack, we've released `netlify-cli@8.0.0` with version range `^12.20.0 || ^14.13.1 || >=16.0.0`.
Updating this PR accordingly.